exercises/scrabble-score/uScrabbleScoreExample.pas

```diff
@@ -0,0 +1,32 @@
+unit uScrabbleScore;
+
+interface
+
+uses
+  System.Generics.Collections;
+
+type
+  TScrabble = class
+  private
+    const
+      FValues : array ['A' .. 'Z'] of integer =
+        (1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10);
+  public
+    class function Score(AInp : string) : integer;
+  end;
```

Comment:
You don't have to make this change if you don't want to, this is just an example solution.  I just wanted to show you an alternative way to write this that is a little cleaner.
```pascal
type
  TScrabble = class
  private
    type
      TAlphaVals = array['A'..'Z'] of integer;
    const
      FValues : TAlphaVals = (1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10);
  public
    class function Score(AInp : string) : integer;
  end;
```